src/store/state/state-model.ts

```diff
@@ -40,13 +42,14 @@ export const stateModel: StateModelIF = {
       isValidCompanyProvisions: true,
       isValidResolutionDate: true
     },
-    validFlags: {
+    flagsReviewCertify: {
+      // NB: this must be in same order as ComponentsReviewCertify enum!
       isValidEffectiveDate: true,
-      isValidFileNum: true, // Staff only
```

Comment:
This is a bugfix: there are 2 flags for the same component here (isValidFileNum and isValidCourtOrder are both for 'court-order' component). So I removed FileNum.

Reply:
^^ What this did is cause a mismatch between the valid flags list and the components list.